roxie/ccd/ccdserver.cpp

```diff
@@ -9138,10 +9153,12 @@ class CRoxieServerThroughSpillActivity : public CRoxieServerActivity
             }
             catch (...)
             {
-                IException *E = MakeStringException(ROXIE_INTERNAL_ERROR, "Unknown exception caught in CRoxieServerThroughSpillActivity::start");
+                IException *E = MakeStringException(ROXIE_INTERNAL_ERROR, "Unknown exception caught in CRoxieServerThroughSpillActivity::doStart");
```

Comment:
In this case I don't think the message should change.  One of only a few places where you still override start..